services/src/main/java/org/fao/geonet/services/resources/DownloadArchive.java

```diff
@@ -347,9 +347,9 @@ private void notifyAndLog(boolean doNotify, String id, String uuid, String acces
         if (doNotify) {
 
             String site = sm.getSiteId();
-            String host = sm.getValue("system/feedback/mailServer/host");
-            String port = sm.getValue("system/feedback/mailServer/port");
-            String from = sm.getValue("system/feedback/email");
+            String host = sm.getValue("catalog/feedback/mailServer/host");
```

Comment:
If renaming the keys then I don't believe we should be using feedback for smtp configuration.  My assumption is that the feedback feature was the first usage of the smtp settings.

As the smtp server is used for sending emails throughout the system - (i.e. approval emails in the workflow) , we may want to take the opportunity to change the key so that it does not have "feedback" in the name.